### PR TITLE
Fix build with BUILD_LAME disabled

### PR DESCRIPTION
**Pull request by Qantas94Heavy**

### CODE REVIEW

File: mscore/musescore.cpp [hunk 1]
```diff
@@ -6685,6 +6685,11 @@ bool MuseScore::canSaveMp3()
 
 bool MuseScore::saveMp3(Score* score, const QString& name)
       {
+#ifndef USE_LAME
```
Comment (anatoly-os) [inline mscore/musescore.cpp]: This code is not needed here. This method redirects call to the `MuseScore::saveMp3(Score* score, QIODevice* device, bool& wasCanceled)`.

File: mscore/musescore.cpp [hunk 2]
```diff
@@ -6701,13 +6706,15 @@ bool MuseScore::saveMp3(Score* score, const QString& name)
...
       return res;
+#endif
       }
 
 bool MuseScore::saveMp3(Score* score, QIODevice* device, bool& wasCanceled)
       {
 #ifndef USE_LAME
       Q_UNUSED(score);
-      Q_UNUSED(name);
+      Q_UNUSED(device);
```
Comment (anatoly-os) [inline mscore/musescore.cpp]: Good catch! Thank you.

**Comment by anatoly-os** [comment]
@Qantas94Heavy thank you for the first contribution. Did you sig CLA (https://musescore.org/en/cla)?

**Comment by Qantas94Heavy** [comment]
@anatoly-os I believe I've already signed CLA for previous contribution?

Anyway I've removed the change to the wrapper function.

**Pull request merged**